maidr/core/maidr.py

```diff
@@ -179,3 +182,21 @@ def _inject_plot(plot: HTML, maidr: str) -> Tag:
             ),
             tags.script(maidr),
         )
+
+        if Environment.is_interactive_shell():
+            # If running in an interactive environment (e.g., Jupyter Notebook),
+            # display the HTML content using an iframe to ensure proper rendering
+            # and interactivity. The iframe's height is dynamically adjusted
+            base_html = tags.iframe(
+                srcdoc=str(base_html.get_html_string()),
+                width="100%",
+                height="100%",
+                scrolling="auto",
+                style="background-color: #fff",
+                frameBorder=0,
+                onload="""
+                    this.style.height = this.contentWindow.document.body.scrollHeight + 100 + 'px';
```

Comment:
Hey Daksh,

When I worked on the py-shiny renderer, I faced a UI/UX issue when displaying the generated MAIDR plot on the shiny component. The major problem with the behaviour I observed on Shiny was that the overflow properties were triggered if pre-defined styles were used. 

Can you share your findings on how Python notebooks renders the cells? After looking at the documentation, I can see that style customizations are allowed for Python notebooks. Does overflow get applied to the plot if pre-defined styles are employed?

Regards,
Krishna Anandan Ganesan.